django-verdant/rca_ee/models.py

```diff
@@ -107,10 +107,16 @@ def serve(self, request):
         context['form'] = form
         return render(
             request,
-            self.template,
+            self.get_template(request),
             context
         )
 
+    def get_template(self, request, *args, **kwargs):
+        if request.is_ajax() and request.GET.get('format') == 'enquiry_form':
```

Comment:
I think ATS will just ignore that though, and serve it from its cache anyway. That setting only makes sure that the ajax request skips the browser cache.